apps/dotcom-worker/src/lib/worker.ts

```diff
@@ -87,14 +87,15 @@ const Worker = {
 }
 
 export function isAllowedOrigin(origin: string) {
+	if (env.IS_LOCAL === 'true') return true
```

Comment:
@SomeHats wait, doesn't this need `env: Environment` also passed to it...? 😬

Reply:
yikes good catch. let's add a lint rule to prevent global `env` access for the worker package

Reply:
i tried that but it's not global - it's actually imported at the top is why it's not complaining 🤦

Reply:
also, it looks tricky because the `createCors` also calls this and I don't think it has access to the `env` but I can't quite tell 🤔

Reply:
oh my bad, this is left over from an earlier attempt at this and isn't needed - sorry! will remove

Reply:
ah no worries! cool